src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyQueueCapacityAction.java

```diff
@@ -30,7 +30,7 @@
 public class ModifyQueueCapacityAction implements Action {
 
   public static final String NAME = "modify_queue_capacity";
-  public static final int COOL_OFF_PERIOD_IN_SECONDS = 300;
+  public static final long COOL_OFF_PERIOD_IN_MILLIS = 300 * 1_000_000;
```

Comment:
Do you want milliseconds or microseconds?

Reply:
Fixed